Support.md

```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
+- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately, and dealing with the consequences along the way.
 - I know how to mentor a customer to the best path, and I understand that the customer's stated issue and the actual issue might not be the same thing
+- I can confidently and charismatically pitch ideas, positively influencing and convincing people to take decisive action based on reaching consensus.
+- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
 ### I lead technically
-
 - I am trusted across Octopus Deploy and have established good working relationships across the organization, and I co-ordinate our response to complicated support issues by pulling in the right people and teams at the right time.
 - I actively research and implement technical solutions to improve our response times
-- I confidently, accurately, and quickly replicate complex deployment scenarios to assist with issue replication and resolution 
+- I confidently, accurately, and quickly replicate complex deployment scenarios to assist with issue replication and resolution.
+-  I recognise problems and proactively engage with other people to prevent or resolve them quickly.
 
 ### I am a mentor of mentors
-
 - I know when to directly assist with a ticket, and when to encourage others to learn more.
-- I encourage others to find creative solutions to problems and accept that my way isn't the only way.
-- I help others in my team to become mentors.
+- I am actively involved in the retrospective process, and I am comfortable challenging and coaching my peers. I encourage others to find creative solutions to problems and accept that my way isn't the only way.
 - I am always on the lookout for opportunities to teach and show others alternate, cleaner ways to delight our customers in an ego-less manner. 
 - I am working to improve our processes, making it easier for everyone in our team to grow and learn.
+- I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles, making sure others understand the solution throughout the process.
 
-### Everything is a learning oppurtunity
-
+### Everything is a learning opportunity
 - I am constantly learning new technologies and methodologies that relate to Octopus.
 - I work with teams outside of Support to further enhance my knowledge
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 - I share the lessons I learn so the entire team and Octopus can benefit
+
+
+
+## Principal Support Engineer
+
+_I am an influential Support Leader, a person who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Support to deliver significant lasting change across Octopus._
```

Comment:
I like this distinction. Really helps explain the influence component of thinking and acting in a much broader way.